pvlib/tests/iotools/test_bsrn.py

```diff
@@ -85,7 +85,7 @@ def test_get_bsrn(expected_index, bsrn_credentials):
         station='tam',
         username=username,
         password=password,
-        local_path='')
+        save_path='')
```

Comment:
What happens when `save_path=''`? An empty string is not `None` (as the function logic tests for). Is this supposed to save a file?